docs/file_formats.md

```diff
@@ -11,6 +13,14 @@ When you're ready to use Casanovo for *de novo* peptide sequencing, you can inpu
 All three of the above file formats can be used as input to Casanovo for *de novo* peptide sequencing.
 As the official PSI standard format containing the complete information from a mass spectrometry run, mzML should typically be preferred.
 
+### Model Weights
+
+In addition to MS/MS spectra, Casanova also optionally accepts a model weights (.ckpt) input file when running in training, sequencing or evaluating mode.
+If no input weight file is provided, Casanovo will automatically use the most recent compatible weights from the official Casanovo github repo, which will be downloaded and cached if they are not already.
```

Comment:
**todo:** We should add a little bit more explanation about the resolving logic for compatible weights (cfr. thread on Slack with Mike Riffle).